source/common/router/rds_impl.h

```diff
@@ -49,75 +49,5 @@ class StaticRouteConfigProviderImpl : public RouteConfigProvider {
   ConfigConstSharedPtr config_;
 };
 
-/**
```

Comment:
@junr03 to make reviewing this easier, can you please split out the file move into a separate PR with no code changes. Then do the PR with the changes? It's really hard to tell what changed. Or just don't move things in this PR and clean it up in another PR. Thanks.